plugins/feeds/public/otx_alienvault.py

```diff
@@ -88,22 +85,27 @@ def analyze(self, item):
 
                 obs.tag(tags)
                 obs.add_context(self.name, context)
-
+                investigation.link_to(obs, "Observed", "OTXAlienVault")
             elif type_ind in entity.EntityType:
-                entity.Entity(
+                ent = entity.Entity(
                     name=otx_indic["indicator"],
                     type=self._TYPE_MAPPING.get(otx_indic["type"]),
                 ).save()
-
+                investigation.link_to(ent, "Observed", "OTXAlienVault")
             elif type_ind in indicator.IndicatorType:
                 if type_ind == indicator.IndicatorType.yara:
-                    indicator.Indicator(
+                    ind_obj = indicator.Indicator(
                         name=f"YARA_{otx_indic['indicator']}",
-                        pattern=otx_indic["content"],
+                        pattern="OTX",
                         type=indicator.IndicatorType.yara,
                         location="OTX",
                         diamond=indicator.DiamondModel.capability,
-                    ).save()
+                    )
+                    # sometimes the content is empty
+                    if otx_indic["content"]:
+                        ind_obj.pattern = otx_indic["content"]
+                        ind_obj.save()
+                        investigation.link_to(ind_obj, "Observed", "OTXAlienVault")
```

Comment:
If we're only saving the indicator if it has a `"content"` key, then we might as well just do something like

```python
if not otx_indic['content']:
  continue  # maybe also log a message?
ind_obj = indicator.Indicator(
...
pattern=otx_indic['content'],
...
).save()
```